rust-runtime/aws-smithy-runtime/src/client/http/body/minimum_throughput/options.rs

```diff
@@ -87,7 +87,7 @@ pub struct MinimumThroughputBodyOptionsBuilder {
     grace_period: Option<Duration>,
 }
 
-const DEFAULT_GRACE_PERIOD: Duration = Duration::from_secs(0);
+const DEFAULT_GRACE_PERIOD: Duration = Duration::from_secs(20);
```

Comment:
**correctness**: Is this the right one?

I also see https://github.com/smithy-lang/smithy-rs/blob/22a0e3033bfc7834e6ad09a902df37cc9d1d82f7/rust-runtime/aws-smithy-runtime-api/src/client/stalled_stream_protection.rs#L16

Reply:
Whats the difference between these two? 🤔 , confusing that we have two constants for this..

Reply:
ah nuts, no wonder. Let's delete one.